python/nano/src/bigdl/nano/deps/neural_compressor/onnx/pytorch/quantization.py

```diff
@@ -58,9 +58,5 @@ def collate_fn(batch):
         return model, calib_dataloader, metric
 
     def _post_execution(self, q_model):
-        # TODO Don't save, directly use q_model to create runtime
-        with TemporaryDirectory() as dir:
-            saved_onnx = Path(dir) / 'tmp.onnx'
-            q_model.save(saved_onnx)
-            return PytorchONNXRuntimeModel(str(saved_onnx),
-                                           onnxruntime_session_options=self.session_options)
+        return PytorchONNXRuntimeModel(q_model.model,
+                                       onnxruntime_session_options=self.session_options)
```

Comment:
As shown in INC quantize with onnxruntime, we don't need to save the quantized model.